scanapp/tasks.py

```diff
@@ -120,93 +176,103 @@ def save_results_to_db(scan_id, json_data, scan_type, URL, folder_name):
             total_errors = total_errors + 1
 
     scan_result = insert_into_db.insert_into_save_results(
-        code_info = code_info,
-        scanned_json_result = json_data,
-        scanned_html_result = '<h4>' + str(json.dumps(json_data)) + '</h4>',
-        scancode_notice = json_data['scancode_notice'],
-        scancode_version = json_data['scancode_version'],
-        files_count = json_data['files_count'],
-        total_errors = total_errors,
-        scan_time = 5000,
+        code_info=code_info,
+        scanned_json_result=json_data,
+        scanned_html_result='<h4>' + str(json.dumps(json_data)) + '</h4>',
+        scancode_notice=json_data['scancode_notice'],
+        scancode_version=json_data['scancode_version'],
+        files_count=json_data['files_count'],
+        total_errors=total_errors,
+        scan_time=5000,
     )
```

Comment:
Same comments as above. If there is a need for formatting, do not mix it up in the same unrelated commit.